templates/gen_component/lib/components/component.ex.eex

```diff
@@ -17,7 +17,6 @@ defmodule <%= @app_module_name %>.Components.<%= @component_module_name %> do
     def verify( text ) when is_bitstring(text), do: {:ok, text}
     def verify( _ ), do: :invalid_data
 
-    @impl Scenic.Component
```

Comment:
Oh, I though that those should be added, but for `Scenic.Scene`. If remember correctly `use Scenic.Component` adds `use Scenic.Scene`.

Reply:
![image](https://user-images.githubusercontent.com/1707779/107070389-e50e0500-67a8-11eb-9640-6bc7a0e06001.png)

Reply:
(agreed)